src/Sarif.Converters/ToolFormatConverter.cs

```diff
@@ -60,44 +60,19 @@ public class ToolFormatConverter
                 using (var outputTextWriter = new StreamWriter(outputTextStream))
                 using (var outputJson = new JsonTextWriter(outputTextWriter))
                 {
-                    if (conversionOptions.HasFlag(ToolFormatConversionOptions.PrettyPrint))
+                    if (loggingOptions.Includes(LoggingOptions.PrettyPrint))
                     {
                         outputJson.Formatting = Formatting.Indented;
                     }
 
                     using (var output = new ResultLogJsonWriter(outputJson))
                     {
-                        ConvertToStandardFormat(toolFormat, input, output, pluginAssemblyPath);
+                        ConvertToStandardFormat(toolFormat, input, output, loggingOptions, pluginAssemblyPath);
                     }
                 }
             }
         }
 
-        /// <summary>Converts a tool log file to the SARIF format.</summary>
-        /// <param name="toolFormat">The tool format of the input file.</param>
-        /// <param name="inputFileName">The input log file name.</param>
-        /// <param name="outputFileName">The name of the file to which the resulting SARIF log shall be
-        /// written. This cannot be a directory.</param>
-        /// <remarks>This overload should be used only from test code.</remarks>
-        public void ConvertToStandardFormat(
```

Comment:
>ConvertToStandardFormat [](start = 20, length = 23)

Yay for deletion

Reply:
Yay! Death to unnecessary overload

---
In reply to: [136628767](https://github.com/Microsoft/sarif-sdk/pull/614#discussion_r136628767) [](ancestors = 136628767)